src/Moq/Properties/Resources.Designer.cs

```diff
@@ -87,6 +87,15 @@ internal class Resources {
 			}
 		}
 		
+		/// <summary>
+        ///   Looks up a localized string similar to CallBase cannot be used to mock Delegate types..
+        /// </summary>
```

Comment:
There appears to be something wrong with indentation here, could you please take a look & fix this?